README.rst

```diff
@@ -107,11 +256,6 @@ Configuration
 Operations
 ==========
 
-Bootstrapping Consumers
------------------------
-
-.. todo:: Rewrite this as part of the tutorial using pgbench after the replication worker is done.
-
```

Comment:
Oh wasn't saying putting everything into a single image or anything, just adding a demo one with just some helper shell scripts or commands or something, not sure - like I said I hadn't worked through it 100%.

I do like the bottled water docs you linked to, will try to model it on that.